stable/kommander-ui/templates/hooks-roles.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Will this SA always be needed, or should we have it's creation flagged?

Reply:
this may be an existing problem with the chart? but also not in the scope of this PR. Could we address this on a later pr?

Reply:
:thinking: It seems like this is something we may wanna work on in this scope yeah? Set the baseline that it's optional?

Reply:
I will check, but doesnt seem like something i want to touch with this many moving parts.

Reply:
this is required as far as i can tell, by the deployment.